specs/proposals.md

```diff
@@ -24,13 +26,23 @@ are part of later specification milestones.
 
 [cannon]: https://github.com/ethereum-optimism/cannon
 
-## Proposing L2 output commitments
+## Constants
+
+| Name                   | Value  |
+| ---------------------- | ------ |
+| `SUBMISSION_FREQUENCY` | `100?` |
```

Comment:
It actually annoys me haha, because I have no context to understand them.

But it's great to have them listed together somewhere, for reference!

Suggestion:
- move this to the end
- recall the default value in the text, so that someone that reads the spec does not have to scroll up/down to see what the expected value is

Reply:
I'd be onboard with this. I'd like to review the full doc for this kind of thing, which I'm tracking here and you might wish to comment on: #252

Reply:
> * move this to the end

Actually, I'd prefer to keep it at the top, might as well follow EIP convention.